pkgs/top-level/all-packages.nix

```diff
@@ -24465,8 +24465,6 @@ with pkgs;
     noto-fonts-emoji-blob-bin
     noto-fonts-extra;
 
-  nuclear = callPackage ../applications/audio/nuclear { };
```

Comment:
Technical hint - while I do not endorse this PR for reasons mentioned by others, it should still be correct if it should any chance of merging.
Please add a `throw` to aliases.nix that mentions the removal of the package.